src/test/scala/mesosphere/marathon/integration/setup/ProcessOutputToLogStream.scala

```diff
@@ -1,13 +1,40 @@
 package mesosphere.marathon
 package integration.setup
 
-import org.slf4j.LoggerFactory
+import org.slf4j.{ LoggerFactory, Logger }
 
 import scala.sys.process.ProcessLogger
 
-case class ProcessOutputToLogStream(process: String) extends ProcessLogger {
-  val log = LoggerFactory.getLogger(s"mesosphere.marathon.integration.process.$process")
-  override def out(msg: => String): Unit = log.debug(msg)
-  override def err(msg: => String): Unit = log.warn(msg)
+final case class ProcessOutputToLogStream(process: String, maxLinesPerPeriod: Int = 100, periodMillis: Long = 1000L) extends ProcessLogger {
```

Comment:
Why not use `FiniteDuration` for `period`?

Reply:
It could; I was wanting to avoid converting a duration to milliseconds on every call to `rateLimited`; it is logging code and performance here is more critical.

Reply:
But ¯\\\_(ツ)\_/¯ not a big deal to just pull out the millis in the constructor. :) ✅